agent/cache-types/config_entry.go

```diff
@@ -79,18 +79,22 @@ func (c *ConfigEntry) Fetch(opts cache.FetchOptions, req cache.Request) (cache.F
 	reqReal.QueryOptions.MaxQueryTime = opts.Timeout
 
 	// Always allow stale - there's no point in hitting leader if the request is
-	// going to be served from cache and endup arbitrarily stale anyway. This
+	// going to be served from cache and end up arbitrarily stale anyway. This
 	// allows cached service-discover to automatically read scale across all
```

Comment:
```suggestion
	// allows cached config entries to automatically read scale across all
```

Need to fix this copy and paste error everywhere.